react-carousel/my-app/src/App.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+import { Buttons } from './Buttons';
+import {
+  FaChevronCircleLeft,
+  FaChevronCircleRight,
+  FaCircle,
+  FaRegCircle,
+} from 'react-icons/fa';
+import { ImageDisplay } from './ImageDisplay';
+import '../public/images/fushiguro.webp';
+import '../public/images/inumaki.webp';
+import '../public/images/itadori.webp';
+import '../public/images/kugisaki.webp';
+import '../public/images/panda.webp';
+import '../public/images/zen-in.webp';
+
+function App() {
```

Comment:
It would be better to put all this into a Carousel component. That would make it reusable.